client/app/iplb/vrack/iplb-vrack.html

```diff
@@ -27,17 +27,25 @@
                 </span>
             </dd>
         </dl>
-        <cui-action-button class="iplb-vrack__box-button"
-            data-ng-if="$ctrl.actions.activateVrack.isAvailable()"
-            data-action="$ctrl.actions.activateVrack"
-            data-type="secondary"></cui-action-button>
-        <cui-action-button class="iplb-vrack__box-button"
-            data-ng-if="$ctrl.actions.deActivateVrack.isAvailable()"
-            data-action="$ctrl.actions.deActivateVrack"
-            data-type="secondary"></cui-action-button>
+        <oui-button data-ng-if="$ctrl.actions.activateVrack.isAvailable()"
+            data-text="{{::$ctrl.actions.activateVrack.text}}" 
+            data-on-click="$ctrl.actions.activateVrack.callback()"
+            data-disabled="$ctrl.actions.activateVrack.isAvailable && !$ctrl.actions.activateVrack.isAvailable()"
```

Comment:
This button will never be disabled, if it's displayed `$ctrl.actions.activateVrack.isAvailable()` is true therefore `data-disabled` will always be false, so it might not be necessary

Reply:
Good catch